### PR TITLE
exercise set 3

### PR DESCRIPTION
**Pull request by brosekerg**

### CODE REVIEW

File: exercise 3-2.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+mysql> insert into visit values ('Rowdy', 'Roundhouse', 'roundhouser@dewv.edu', 'Freshman', 'Residential', 'Philosophy', 'Basketball', 'Steve', 'Stauber', CURRENT_TIMESTAMP, NULL, 'SC413', 'Writing Center', 'Yes', 'Yes', 'long statements');
+ERROR 1406 (22001): Data too long for column 'purpose_achieved' at row 1
+mysql> insert into visit values ('Rowdy', 'Roundhouse', 'roundhouser@dewv.edu', 'Freshman', 'Residential', 'Philosophy', 'Basketball', 'Steve', 'Stauber', CURRENT_TIMESTAMP, NULL, 'SC413', 'Writing Center', 'Y', 'Y', 'long statements');
```
Comment (smattingly) [inline exercise 3-2.txt:3]: You didn't need to put `CURRENT_TIMESTAMP` in your statement; it is the default value for the column so it is populated automatically.

**Comment by brosekerg** [comment]
the reason i put current timestamp was because i thought it would be easier than writing out the time and potentially messing up the query
________________________________

**Comment by smattingly** [comment]
Your comment shows me you are missing the point-- your `INSERT` does not need `CURRENT_TIMESTAMP` **or** to spell out the date/time. The default specified in the table definition will automatically put the current date/time into the field if your `INSERT` specifies *no value at all*.

**Comment by brosekerg** [comment]
Okay thats fine so what is the need for a current timestamp statement if that is the case? Just one of them things that have become outdated?



Sent via the Samsung Galaxy S9, an AT&T 5G Evolution capable smartphone



-------- Original message --------

**Comment by brosekerg** [comment]
But yes I understand. Thank you!



Sent via the Samsung Galaxy S9, an AT&T 5G Evolution capable smartphone



-------- Original message --------

**Comment by smattingly** [comment]
It's needed to define the default value when the table is created. It's not needed on `INSERT` because the default has been defined.

**Comment by brosekerg** [comment]
Aha yes that makes perfect sense



Sent via the Samsung Galaxy S9, an AT&T 5G Evolution capable smartphone



-------- Original message --------

**Pull request merged**